index.d.ts

```diff
@@ -2033,6 +2058,8 @@ declare namespace cytoscape {
         includeTargetLabels?: boolean | undefined;
         /** A boolean indicating whether to include overlays (such as the one which appears when a node is clicked) in the bounding box (default true). */
         includeOverlays?: boolean | undefined;
+        /** A boolean indicating whether to include underlays in the bounding box (default: true). */
+        includeUnderlays?: boolean | undefined;
```

Comment:
This is a minor style nitpick -- when you have optional properties, the general style convention is that you don't need to include the `| undefined` part.  I noticed that some linces in this community file follow that convention and some don't.

Reply:
OK, let's remove the `| undefined` parts